tests/spec_decode/test_multi_step_worker.py

```diff
@@ -477,6 +481,106 @@ def test_multi_step_with_batch_expansion_incorrect_output():
     assert (num_mismatch > 0)
 
 
+@torch.inference_mode()
+@pytest.mark.parametrize('num_steps', [1, 2, 3, 4])
+@pytest.mark.parametrize('attn_backend',
+                         [_Backend.XFORMERS, _Backend.FLASH_ATTN])
```

Comment:
nit : Can you please add a comment saying that the choice of backends forces the multi_step_worker to choose between the vanilla model_runner and TP1DraftModelRunner and that we can test both code paths?